js/repeater.js

```diff
@@ -691,90 +726,71 @@
 			}
 		},
 
-		runRenderer: function (viewTypeObj, data, callback) {
-			var $container, i, l, response, repeat, subset;
-
-			function addItem ($parent, resp) {
-				var action;
-				if (resp) {
-					action = (resp.action) ? resp.action : 'append';
-					if (action !== 'none' && resp.item !== undefined) {
-						$parent = (resp.container !== undefined) ? $(resp.container) : $parent;
-						$parent[action](resp.item);
-					}
-				}
-			}
-
+		renderItems: function renderItems (viewTypeObj, data, callback) {
 			if (!viewTypeObj.render) {
 				if (viewTypeObj.before) {
-					response = viewTypeObj.before.call(this, {
+					var addBefore = viewTypeObj.before.call(this, {
 						container: this.$canvas,
 						data: data
 					});
-					addItem(this.$canvas, response);
+					addItem(this.$canvas, addBefore);
 				}
 
-				$container = this.$canvas.find('[data-container="true"]:last');
+				var $container = this.$canvas.find('[data-container="true"]:last');
 				$container = ($container.length > 0) ? $container : this.$canvas;
 
 				if (viewTypeObj.renderItem) {
-					repeat = viewTypeObj.repeat || 'data.items';
+					var subset;
+					var repeat = viewTypeObj.repeat || 'data.items';
```

Comment:
Could we do something like `viewTypeObjItems`. Not a big deal, but there's a lot going on and was struggling with the meaning of `repeat`